husky_components/datagrid/decorators/tiles-view.js

```diff
@@ -0,0 +1,226 @@
+            $container.append(this.$el);
+        },
+
+        /**
+         * Renders the add new tile and binds its click event
+         */
+        renderAddNewTile: function() {
+            var $addNew = $(_.template(templates.addNewTile, {
+                icon: this.options.addNewIcon,
+                title: this.sandbox.translate(this.options.translations.addNew)
+            }));
+
+            $addNew.on('click', function() {
+                this.sandbox.emit(ADD_CLICKED.call(this));
+            }.bind(this));
+
+            this.$el.append($addNew);
+        },
+
+        /**
+         * Renders a single record into the view
+         *
+         * @param {Object} record The data of the record
+         */
+        renderRecord: function(record) {
+            record = processContentFilters.call(this, record);
+            var title = this.getTitleString(record),
+                description = this.getDescriptionSring(record);
+            this.$tiles[record.id] = $(_.template(templates.tile, {
+                icon: this.options.icon,
+                title: title,
+                croppedTitle: this.sandbox.util.cropMiddle(this.getTitleString(record), 20),
+                description: this.sandbox.util.cropMiddle(description, 20)
+            }));
+
+            this.bindTileDomEvents(record.id);
+            this.$el.append(this.$tiles[record.id]);
+        },
+
+        /**
+         * Binds dom events for a single tile
+         *
+         * @param {Object} tileId The id of the tile to bind the events for
+         */
+        bindTileDomEvents: function(tileId) {
+            this.$tiles[tileId].on('click', function() {
+                this.datagrid.itemAction.call(this.datagrid, tileId);
+            }.bind(this));
+        },
+
+        /**
+         * Constructs and returns the title string to render
+         *
+         * @param {Object} record the data to construct the title from
+         * @return {String} the constructed title
+         */
+        getTitleString: function(record) {
+            var title = '';
+            this.options.fields.title.forEach(function(titleField) {
+                title += record[titleField] + ' ';
+            });
+
+            return title.trim();
+        },
+
+        /**
+         * Constructs and returns the description string to render
+         *
+         * @param {Object} record the data to construct the description from
+         * @return {String} the constructed description
+         */
+        getDescriptionSring: function(record) {
+            var description = '';
+            this.options.fields.description.forEach(function(descriptionField) {
+                description += record[descriptionField] + ' ';
+            });
+
+            return description.trim();
+        },
+
+        /**
+         * Destroys the view
+         */
+        destroy: function() {
+            this.$el.remove();
+            this.setVariables();
```

Comment:
Why do we need that call here? Asking because we could inline the method in the `initialize` method otherwise.